llarp/crypto/crypto.cpp

```diff
@@ -1,26 +1,13 @@
 #include "crypto.hpp"
 
-#include <sodium/core.h>
-#include <sodium/crypto_generichash.h>
-#include <sodium/crypto_sign.h>
-#include <sodium/crypto_scalarmult.h>
-#include <sodium/crypto_scalarmult_ed25519.h>
-#include <sodium/crypto_stream_xchacha20.h>
-#include <sodium/crypto_core_ed25519.h>
-#include <sodium/crypto_aead_xchacha20poly1305.h>
-#include <sodium/randombytes.h>
-#include <sodium/utils.h>
```

Comment:
Breaking out the sodium headers this way was deliberate to only include the parts of libsodium we are using, rather than all of libsodium.